docs/Updating-Software-in-Homebrew.md

```diff
@@ -0,0 +1,13 @@
+# Updating Software in Homebrew
+
+Did you find something in Homebrew that wasn't the latest version? You can help yourself and others by submitting a pull request to update the formula.
+
+First, check the pull requests in the [homebrew-core](https://github.com/Homebrew/homebrew-core/pulls) or [linuxbrew-core](https://github.com/Homebrew/linuxbrew-core/pulls) repositories to make sure there isn't already an open PR. You may also want to look through closed pull requests for the formula, as sometimes formulae run into problems preventing them from being updated and it's better to be aware of any issues before putting significant effort into an update.
+
+The guide on [opening a pull request](How-To-Open-a-Homebrew-Pull-Request.md#submit-a-new-version-of-an-existing-formula) should really be all you need, this will explain how to easily change the url to point to the latest version and that's really all you need. If you want to read up on `bump-formula-pr` before using it you could check [the manpage](Manpage.md#bump-formula-pr-options-formula).
```

Comment:
```suggestion
The [How To Open a Homebrew Pull Request](How-To-Open-a-Homebrew-Pull-Request.md) documentation should explain most everything you need to know about the process of creating a PR for a version update. For simple formulae updates, this typically involves changing the URL and sha256.
```

This paragraph still has issues from the previous draft, so this reworks it a bit to integrate it better into the existing text (without going into the lengthy descriptions in the previous suggestion).

It also may be more appropriate to update the "Submit a new version of an existing formula" section of the "How To Open a Homebrew Pull Request" page to integrate a link somewhere to the `bump-formula-pr` manpage. I integrated it more seamlessly in the lengthier suggestion but without that explanation it feels tacked on here (since we're offloading the details onto the PR documentation page).